src/VisualStudio/Setup/UnifiedSettings/csharpIntellisenseSettings.registration.json

```diff
@@ -0,0 +1,246 @@
+{
```

Comment:
It feels like this file should be part of the MS.VS.LanguageServcie.CSharp project

Reply:
It needs to be deployed under /UnifiedSettings. Is this possible to do it if put under MS.VS.LanguageServcie.CSharp? We can address later if it's needed